frontend/src/scenes/experiments/RunningTimeCalculator/RunningTimeCalculatorModal.tsx

```diff
@@ -140,110 +30,39 @@ export function RunningTimeCalculatorModal(): JSX.Element {
-                                onChange={(value) => {
-                                    if (value !== null) {
-                                        setMetricIndex(value)
-                                    }
-                                }}
-                            />
-                        </div>
-
-                        {metricResultLoading ? (
-                            <div className="border-t pt-2">
-                                <div className="h-[100px] flex items-center justify-center">
-                                    <Spinner className="text-3xl transform -translate-y-[-10px]" />
-                                </div>
-                            </div>
-                        ) : (
-                            <div className="border-t pt-2">
-                                {(metric as ExperimentMetric)?.metric_type === ExperimentMetricType.MEAN && (
-                                    <MeanMetricDataPanel />
-                                )}
-                                {(metric as ExperimentMetric)?.metric_type === ExperimentMetricType.FUNNEL && (
-                                    <FunnelMetricDataPanel />
-                                )}
-                            </div>
-                        )}
-                    </div>
-                </div>
+            <EventSelectorStep />
+            {exposureEstimateConfig && <MetricSelectorStep />}
 
+            <div className="deprecated-space-y-6">
                 {!metricResultLoading && uniqueUsers !== null && (
                     <>
-                        {/* Step 2: MDE configuration */}
-                        <div className="rounded bg-light p-4 deprecated-space-y-3">
-                            <div className="flex items-center gap-2">
-                                <span className="rounded-full bg-muted text-white w-6 h-6 flex items-center justify-center font-semibold">
-                                    2
-                                </span>
-                                <h4 className="font-semibold m-0">Choose minimum detectable effect</h4>
-                            </div>
-                            <p className="text-muted">
-                                The minimum detectable effect (MDE) is the smallest improvement you want to be able to
-                                detect with statistical significance. A smaller MDE requires more participants but can
-                                detect subtler changes.
-                            </p>
+                        <RunningTimeCalculatorModalStep
+                            stepNumber={3}
+                            title="Choose minimum detectable effect"
+                            description="The minimum detectable effect (MDE) is the smallest improvement you want to be able to detect with statistical significance. A smaller MDE requires more participants but can detect subtler changes."
```

Comment:
We should make it clear here that it's _relative_ MDE and not absolute (just a suggestion, feel free to reword)
```suggestion
                            description="The minimum detectable effect (MDE) is the smallest relative improvement you want to be able to detect with statistical significance. A smaller MDE requires more participants but can detect subtler changes."
```

Reply:
I went through several iterations of this copy before setting on this one. I'm surprised you only requested one change 😛 

Updated here: 3cfa05fe3ef0477d7a1aa90529636930e616f64f